### PR TITLE
General LOGGING_DISABLED macro prefix adjustment

### PR DESCRIPTION
**Pull request by sergii-vostrikov**
The macro-enabler is now called EDGE_SDK_LOGGING_DISABLED

Relates-to: OLPEDGE-407

Signed-off-by: Sergii Vostrikov <ext-sergii.vostrikov@here.com>

### CODE REVIEW

File: olp-cpp-sdk-core/include/olp/core/logging/Log.h [hunk 1]
```diff
@@ -240,7 +240,7 @@
   }                                         \
   EDGE_SDK_CORE_LOOP_ONCE()
 
-#ifdef LOGGING_DISABLED
+#ifdef EDGE_SDK_LOGGING_DISABLED
```
Comment (andescu) [inline olp-cpp-sdk-core/include/olp/core/logging/Log.h]: please align with coding style
```suggestion
#if defined(EDGE_SDK_LOGGING_DISABLED)
```

File: olp-cpp-sdk-core/include/olp/core/logging/Log.h [hunk 2]
```diff
@@ -323,7 +323,7 @@
 #define LOG_ERROR(tag, message) \
   EDGE_SDK_LOG(::olp::logging::Level::Error, tag, message)
 
-#ifdef LOGGING_DISABLED
+#ifdef EDGE_SDK_LOGGING_DISABLED
```
Comment (andescu) [inline olp-cpp-sdk-core/include/olp/core/logging/Log.h]: Same here

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm [hunk 3]
```diff
@@ -91,14 +91,14 @@ - (HttpTaskStatus)run {
     request.HTTPBody = self.body;
   }
 
-#ifndef LOGGING_DISABLED
+#ifndef EDGE_SDK_LOGGING_DISABLED
```
Comment (andescu) [inline olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: Same here
```suggestion
#if !defined(EDGE_SDK_LOGGING_DISABLED)
```

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm [hunk 4]
```diff
@@ -91,14 +91,14 @@ - (HttpTaskStatus)run {
...
   NSMutableString* debugHeaders = [NSMutableString new];
 #endif
 
   for (NSString* key in self.headers.allKeys) {
     NSString* value = self.headers[key];
     [request setValue:value forHTTPHeaderField:key];
-#ifndef LOGGING_DISABLED
+#ifndef EDGE_SDK_LOGGING_DISABLED
```
Comment (andescu) [inline olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: same

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm [hunk 5]
```diff
@@ -111,7 +111,7 @@ - (HttpTaskStatus)run {
   // Cache the task id for fast retrieve later on
   _httpClient.idTaskMap[@(self.dataTask.taskIdentifier)] = self;
 
-#ifndef LOGGING_DISABLED
+#ifndef EDGE_SDK_LOGGING_DISABLED
```
Comment (andescu) [inline olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: same

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm [hunk 6]
```diff
@@ -162,7 +162,7 @@ - (void)didCompleteWithError:(NSError*)error {
 }
 
 - (void)didReceiveResponse:(NSURLResponse*)response {
-#ifndef LOGGING_DISABLED
+#ifndef EDGE_SDK_LOGGING_DISABLED
```
Comment (andescu) [inline olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: same

File: olp-cpp-sdk-core/include/olp/core/logging/Log.h (hunk 1 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-core/include/olp/core/logging/Log.h]: The scope of this change is to add prefix to the macro.
Changes to coding style should be present in another ticket.

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm (hunk 3 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: ditto

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm (hunk 4 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: ditto

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm (hunk 5 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: ditto

File: olp-cpp-sdk-core/src/network/ios/HttpTask.mm (hunk 6 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-core/src/network/ios/HttpTask.mm]: ditto

File: olp-cpp-sdk-core/include/olp/core/logging/Log.h (hunk 2 above)
Comment (sergii-vostrikov) [reply olp-cpp-sdk-core/include/olp/core/logging/Log.h]: ditto

**Pull request merged**